app/services/allocation_events/params_validator.rb

```diff
@@ -4,9 +4,10 @@ module AllocationEvents
   class ParamsValidator
     include ActiveModel::Validations
 
+    # TODO: remove event_name when allocation `events` endpoint is no longer in use
     attr_reader :event_name, :timestamp
 
-    validates :event_name, inclusion: %w[cancel], presence: true
+    validates :event_name, inclusion: %w[cancel], allow_nil: true
```

Comment:
`event_name` only really makes sense in the context of the previous `events` approach - so if the event name is blank this validator will pass the params, but we'll still get a bad request error raised in the `events` method of allocation events controller. Actually I'm pretty sure validation of `event_name` could be removed now if preferred as the validation occurs in two places. See: https://github.com/ministryofjustice/hmpps-book-secure-move-api/pull/505/files#diff-cf52449dc55f39cafad70842929410deR29

Reply:
yep fine to remove as its being caught elsewhere 👍 
... however you may need a validator for `type` though?

Reply:
There's only one event at the moment, so no need for `type` to be introduced yet. I'd prefer to add that when the next event is added, or at least after we get rid of `event_name`.